LEGO1/mxdssource.cpp

```diff
@@ -0,0 +1,14 @@
+#include "mxdssource.h"
+
+// OFFSET: LEGO1 0x100bffd0
+void MxDSSource::SomethingWhichCallsRead(void* pUnknownObject)
```

Comment:
In the absence of a better name, it might be better to keep it generic like `FUN_100bffd0`. I feel like vague naming could complicate things in situations where a lot of functions appear to do very similar things.

Reply:
I agree, except in the case of virtual methods like this one.

IMO having as much info as possible there for virtual methods makes sense because you can't easily see / refine their naming later through XREFs in Ghidra.

Reply:
Agree with stravant. This is a method in local namespace, not globally visible.
So there's no chance on collision.